test/views/declaration/DeclarationHolderViewSpec.scala

```diff
@@ -82,13 +82,11 @@ class DeclarationHolderViewSpec extends UnitViewSpec with CommonMessages with St
 
       "display empty input with label for EORI" in {
 
-        document.getElementById("eori-label").text() mustBe messages("supplementary.declarationHolder.eori")
+        document.getElementsByClass("govuk-label govuk-label--m").get(1).text() mustBe messages("supplementary.declarationHolder.eori")
```

Comment:
That looks fragile way of getting the element.   I think that is an input field label you are trying to find.
`document.getElementsByAttributeValue("for", "eori").text()` should work better and as an added bonus tests that the label is correctly "for" the input field.

Reply:
Good hint